static/src/js/containers/DownloadHistoryContainer/__tests__/DownloadHistoryContainer.test.js

```diff
@@ -1,89 +1,368 @@
 import React from 'react'
-import Enzyme, { shallow } from 'enzyme'
-import Adapter from '@wojtekmaj/enzyme-adapter-react-17'
-
-import actions from '../../../actions'
-import {
-  DownloadHistoryContainer,
-  mapDispatchToProps,
-  mapStateToProps
-} from '../DownloadHistoryContainer'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
+import { MemoryRouter } from 'react-router-dom'
+
+import * as deployedEnvironment from '../../../../../../sharedUtils/deployedEnvironment'
+import * as getApplicationConfig from '../../../../../../sharedUtils/config'
+
 import { DownloadHistory } from '../../../components/DownloadHistory/DownloadHistory'
+import { mapStateToProps, mapDispatchToProps } from '../DownloadHistoryContainer'
+import * as errorsModule from '../../../actions/errors'
 
-Enzyme.configure({ adapter: new Adapter() })
+jest.spyOn(deployedEnvironment, 'deployedEnvironment').mockImplementation(() => 'prod')
 
-function setup(props) {
-  const enzymeWrapper = shallow(<DownloadHistoryContainer {...props} />)
+jest.spyOn(getApplicationConfig, 'getApplicationConfig').mockImplementation(() => ({
+  env: 'test',
+  defaultCmrPageSize: 20
+}))
 
-  return {
-    enzymeWrapper,
-    props
-  }
-}
+jest.mock('../../../containers/PortalLinkContainer/PortalLinkContainer', () => ({
+  __esModule: true,
+  default: ({ children, to, portalId }) => (
+    <a
+      href={`/downloads/${to.pathname.split('/').pop()}${to.search}`}
+      data-testid="portal-link"
+      data-portal-id={portalId}
+    >
+      {children}
+    </a>
+  )
+}))
 
-describe('mapDispatchToProps', () => {
-  test('onFetchRetrievalHistory calls actions.fetchRetrievalHistory', () => {
-    const dispatch = jest.fn()
-    const spy = jest.spyOn(actions, 'fetchRetrievalHistory')
+const setup = (props) => render(
+  <MemoryRouter>
+    <DownloadHistory {...props} />
```

Comment:
This test is supposed to be testing DownloadHistoryContainer, not DownloadHistory